templates/web/base/admin/users.html

```diff
@@ -26,7 +26,7 @@
         <td>[% PROCESS value_or_nbsp value=user.name %]</td> 
         <td><a href="[% c.uri_for( 'reports', search => user.email ) %]">[% PROCESS value_or_nbsp value=user.email %]</a></td> 
         <td>[% PROCESS value_or_nbsp value=user.from_body.name %]
-            [% IF user.has_body_permission_to('moderate') %] * [% END %]
+            [% IF NOT user.is_superuser AND user.has_body_permission_to('moderate') %] * [% END %]
```

Comment:
Now we have multiple permissions, I wonder if this asterisk would be more use for identifying superusers?